blackbox/docs/protocols/http.txt

```diff
@@ -93,7 +95,8 @@ It is possible to set a default schema while querying the Crate cluster via
 ``_sql`` end point. In such case the HTTP request should contain the
 ``Default-Schema`` header with the specified schema name::
 
-    sh$ curl -sSXPOST '127.0.0.1:4200/_sql?pretty' -H 'Default-Schema: doc' -d '{
+    sh$ curl -H 'Content-Type: application/json' \
+    ... -sSXPOST '127.0.0.1:4200/_sql?pretty' -H 'Default-Schema: doc' -d '{
```

Comment:
okay, I've fixed it up so all of the examples except the first one use heredocs. then I've added this graphic just below the first one:

![image](https://cloud.githubusercontent.com/assets/23469/21135453/3be64cc0-c122-11e6-8696-ba0da3721875.png)

I also reformatted some of the input JSON, and did a few minor copy edits

can you review again please? thanks!